.github/mergify.yml

```diff
@@ -0,0 +1,10 @@
+pull_request_rules:
+- name: label-backend
+  description: Automatically apply backend label
+  conditions:
+    - or:
+      - files~=docs/backend/.*
```

Comment:
I know the `or:` isn't necessary here, but I left it when copying from a different example. I fully expect to add some other paths here soon, like `api-definitions` from #71